src/blk/kernel/KernelDevice.cc

```diff
@@ -538,6 +538,11 @@ void KernelDevice::_discard_update_threads(bool discard_stop)
 
   uint64_t oldcount = discard_threads.size();
   uint64_t newcount = cct->_conf.get_val<uint64_t>("bdev_async_discard_threads");
+  if (newcount == 0) {
```

Comment:
Do we need to clarify that when `bdev_ sync_discard_threads` is greater than 0, `bdev_ sync_discard==false` will be ignored?

Reply:
Parameter description updated.